docs/content/1.getting-started/5.examples.md

```diff
@@ -15,42 +15,7 @@ You can mix and match components to build your own UI.
 
 You can easily build a color mode button by using the `useColorMode` composable from `@nuxtjs/color-mode`.
 
-::component-example
-#default
-:color-mode-button
-
-#code
-```vue [components/ColorModeButton.vue]
-<script setup>
-const colorMode = useColorMode()
-
-const isDark = computed({
-  get () {
-    return colorMode.value === 'dark'
-  },
-  set () {
-    colorMode.preference = colorMode.value === 'dark' ? 'light' : 'dark'
-  }
-})
-</script>
-
-<template>
-  <ClientOnly>
-    <UButton
-      :icon="isDark ? 'i-heroicons-moon-20-solid' : 'i-heroicons-sun-20-solid'"
-      color="gray"
-      variant="ghost"
-      aria-label="Theme"
-      @click="isDark = !isDark"
-    />
-
-    <template #fallback>
-      <div class="w-8 h-8" />
-    </template>
-  </ClientOnly>
-</template>
-```
-::
+:component-example{component="color-mode-button"}
```

Comment:
I think you can let the code here as this component is not in the examples dir: https://ui-r7hwmn6cu-nuxt-js.vercel.app/getting-started/examples#colormodebutton